CMS/Pages Readability Improvements (#380)
octobercms/docs: cms-pages.md

        {% endfor %}
    </ul>

> The default variables and Twig extensions provided by October are described in the [Markup Guide](../markup). During the overall sequence, the handlers are executed as described in the [Dynamic layouts](layouts#dynamic-layouts) article.

Author: @bennothommo I've taken some of your suggestion on this one and just made it more concise.
```suggestion
The default variables and Twig extensions provided by October are described in the [Markup Guide](../markup). The sequence that the handlers are executed in is described by the [Dynamic layouts](layouts#dynamic-layouts) article.
```

    <p>The title of this page is: {{ this.page.title }}</p>

More information can be found at [`this.page` in the Markup guide](../markup/this-page).

<a name="injecting-assets"></a>
## Injecting page assets programmatically

If needed, you can inject assets (CSS and JavaScript files) into pages with the controller's `addCss` and `addJs` methods. It could be done in the `onStart` function defined in the [PHP section](themes#php-section) of a page or [layout](layout) template. Example:

    function onStart()
    {
        $this->addCss('assets/css/hello.css');
        $this->addJs('assets/js/app.js');
    }

> If the path specified in the `addCss` and `addJs` method argument begins with a slash (/), it will be relative to the website root. If the asset path does not begin with a slash, it is relative to the theme.

Reviewer: @DanHarrin IMO, I think this sentence already reads fine as is.

Author: @bennothommo I have to stand by this edit, imagine reading the sentence out loud, you would naturally add the commas to improve flow.

Reviewer: @DanHarrin yeah, that's fair. :)

> Injected assets can be combined by passing them as an array:

Reviewer: @DanHarrin I don't think `in` is required here.
